backends/ebpf/psa/README.md

```diff
@@ -180,6 +180,89 @@ if the ActionProfile is specified for a P4 table.
 **Note:** As of April 2022, support for ActionProfile extern in `psabpf-ctl` CLI/API is not implemented yet. As a workaround
 you can use `psabpf-ctl table` command for this extern.
 
+### ActionSelector
+
+[PSA specification](https://p4.org/p4-spec/docs/PSA-v1.1.0.html#sec-action-selector) is a table implementation similar to
+an ActionProfile, but extends its functionality with support for group of action. If a table entry contains member reference,
+ActionSelector behaves in the same way as ActionProfile. For group references compiler generates additional maps and code
+in order to resolve group reference into member reference. One of additional maps is a map (hash map of maps) that maps
+group reference into inner map (array map) which describes a group. Such map (might be created at runtime by `psabpf-ctl`)
+contains number of members in group in first element, other contain member references. To choose a member from a group,
+checksum is calculated from keys with match type `selector`. Next, obtained member reference from group map is used to
+get action and execute it.
+
+Second created map by compiler is a map containing an action for empty group. For ActionSelector there are two fields
+stored in a table that uses given ActionSelector instance, one is reference, second is marker whether reference points
+to group or member.
```

Comment:
Próbowałem to przeredagować jak poniżej:

```suggestion
A second created map contains an action for an empty group. For the ActionSelector, there are two fields stored in a table that uses given ActionSelector instance, one is reference, second is marker whether reference points to group or member.
```

ale nie mam pojecia co miałeś na myśli w tym zdaniu: `there are two fields stored in a table that uses given ActionSelector instance, one is reference, second is marker whether reference points to group or member.`